### ISSUE TITLE
Question: how should this be used?

### ISSUE THREAD

**Issue by jgoz** [mekwall/normalized-fs#4]
In your [https://github.com/isaacs/node-graceful-fs/pull/131#issuecomment-457036039](comment), you asked for people to be early adopters and I would be happy to be one of them.

How do you envision this package should be used? Ideally, for my own use case, `normalized-fs` could be set up to replace `graceful-fs` without much hassle. I see you have a `replace-gfs` folder in this project - could that be used in other projects?

Perhaps by adding `normalized-fs` as a (dev) dependency and setting up a resolution for `graceful-fs` like you have done, it would avoid pulling `graceful-fs` altogether.

**Comment by mekwall** [comment]
The `replace-gfs` package is only used internally atm but I am thinking about moving it out to it's own package that can be used in the way you describe. Would that help?

**Comment by jgoz** [comment]
Yeah, that could certainly help. Ideally, major projects (like Jest) would drop `graceful-fs` altogether, but I'm thinking pessimistically &mdash; it will take time before it starts disappearing from dependency graphs, so having a standard way of overriding it would be nice for the near term.
